scripts/create_test_subset.py

```diff
@@ -583,14 +583,16 @@ def transfer_participants(
     """Transfers relevant participants between projects"""
     existing_participants = papi.get_participants(target_project)
 
-    target_project_epids = [
-        participant['external_id'] for participant in existing_participants
-    ]
+    target_project_pid_map = {
+        participant['external_id']: participant['id'] for participant in existing_participants
+    }
 
     participants_to_transfer = []
     for participant in participant_data:
-        if participant['externalId'] not in target_project_epids:
+        if participant['externalId'] in list(target_project_pid_map.keys()):
```

Comment:
I'm coming to this late and this isn't a super big deal because it's in auxillary code, but this line turns an O(1) lookup into an O(n). Can just leave it as:

```suggestion
        if participant['externalId'] in target_project_pid_map:
```

Reply:
This is actually really awesome to know, cheers @illusional. I'll make sure thats fixed in the refactoring PR 👍